Tests/Tests.Shared/ConfigurationTests.cs

```diff
@@ -126,23 +143,15 @@ public void UnableToOpenWithNoKey()
             ReliesOnEncryption();
 
             // Arrange
-            var config = new RealmConfiguration("UnableToOpenWithNoKey.realm");
-            Realm.DeleteRealm(config);  // ensure guarded from prev tests
-            var emptyKey = new byte[64];
-            config.EncryptionKey = emptyKey;
-            using (Realm.GetInstance(config))
+            _configuration.EncryptionKey = new byte[64];
```

Comment:
I remember we wanted to make configurations immutable. This will work now but maybe we will have to change it then?